### ISSUE TITLE
Config isn't applied correctly

### ISSUE THREAD

**Issue by belkin** [vaersaagod/matrixmate#39]
Hi

I have an issue with latest version of craft and matrixmate plugin where config isn't applied to sections/entryTypes correctly but '*' works fine.

Craft: 3.7.25.1
Matrixmate: 1.3.1

```
<?php

return [
    'fields' => [
        'contentBlocks' => [
            '*' => [
                'groups' => [[
                    'label' => 'Text and images',
                    'types' => ['text', 'images'],
                ]]
            ],
            'section:courses' => [
                'groups' => [[
                    'label' => 'Text and images',
                    'types' => ['text', 'images'],
                ], [
                    'label' => 'CTA',
                    'types' => ['cta'],
                ]]
            ],
        ],
    ],
];

**Comment by mmikkel** [comment]
Hi @belkin, thanks for reporting. Looks like the issue is isolated to _new entries_ (i.e. an entry that have never been saved) when using `'section:{sectionHandle}'` syntax for the config. 

I'll get a fix in ASAP.

**Comment by mmikkel** [comment]
Should be fixed in [MatrixMate 1.3.2](https://github.com/vaersaagod/matrixmate/releases/tag/v1.3.2). Thanks for reporting!
